src/Compilers/CSharp/Test/Syntax/Parsing/ParserRegressionTests.cs

```diff
@@ -79,28 +77,13 @@ public void c024928()
             var tree = SyntaxFactory.ParseSyntaxTree(test);
         }
 
+#if !DEBUG 
```

Comment:
Also, we must have "release-only" tests and "debug-only" tests, but I couldn't find any with a quick `git grep`.